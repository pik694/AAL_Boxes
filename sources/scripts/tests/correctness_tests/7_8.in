7
65 100 1
2 10 11
4 56 89
68 81 16
19 77 29
53 8 83
58 49 57

