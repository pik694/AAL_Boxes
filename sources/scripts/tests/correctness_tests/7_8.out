1 2 0 5 4 3 6 
354397->354177
